happytransformer/adaptors/albert.py

```diff
@@ -0,0 +1,15 @@
+from transformers import (
+    AlbertForMaskedLM, AlbertTokenizerFast, AlbertForQuestionAnswering,
+    AlbertForSequenceClassification
+)
+from .adaptor import Adaptor
```

Comment:
Why is this in a separate file from the other BERT models?

Reply:
The idea behind separating it is we can tell people that code for a new Adaptor should look like `albert.py`, and they can see directly what is required for a single model. I can put it back with the other BERTs if you want though

Reply:
I see. Yeah put it back please, it's a little confusing. I'm sure as we add models we'll eventually have a standalone Adaptor within a file.